acra/regulations/_index.md

```diff
@@ -2,3 +2,50 @@
+
+_&gt; &quot;Taking into account the state of the art, the costs of implementation and the nature, scope, context and purposes of processing as well as the risk of varying likelihood and severity for the rights and freedoms of natural persons, the controller and the processor shall implement appropriate technical and organisational measures to ensure a level of security appropriate to the risk,...&quot;_
+
+Also, the components of Acra allow encrypting data on the client&#39;s side, which enables secure transfer of the data in encrypted form through an untrusted channel.
+
+## Data protection by design and by default
+
+[Article 25 of GDPR](https://gdpr-info.eu/art-25-gdpr/)
+
+The default settings of Acra are pre-configured in the most secure way to provide that exact &quot;secure by default&quot; state of the system. This is enabled through the default encryption settings for the data transfer between the Acra&#39;s components and the database ( [SSL](http://info.ssl.com/article.aspx?id=10241) and [Secure Session]({{< ref "themis/crypto-theory/cryptosystems/secure-session.md" >}}) are used).
+
+_&gt;&quot;In particular, such measures shall ensure that by default personal data are not made accessible without the individual&#39;s intervention to an indefinite number of natural persons.&quot;_
+
+## Logging, Intrusion Detection, and Notification
+
+[Articles 33](https://gdpr-info.eu/art-33-gdpr/) [and 34 of GDPR](https://gdpr-info.eu/art-34-gdpr/)
+
+Acra makes the communication with the users in case of a data breach easier because the data is encrypted (see Art 34 of GDPR):
+
+_&gt; &quot;The communication to the data subject referred to in paragraph 1 shall not be required if any of the following conditions are met:
+the controller has implemented appropriate technical and organisational protection measures, and those measures were applied to the personal data affected by the personal data breach, in particular, those that render the personal data unintelligible to any person who is not authorised to access it, such as encryption.&quot;_
+
+All the components of Acra are logging the requests/queries/events that take place within Acra plus supports [secure logging]({{< ref "/acra/security-controls/security-logging-and-events/" >}}) that prevents tampering messages, removing, adding or changing the order of log entries. This provides the logs of everything that was happening in Acra in case of a security incident. Using the logs makes it easier to notify the supervisory authority and users about the exact details of a data breach (which complies with the articles [33](http://gdpr-info.eu/art-33-gdpr/) and [34](http://gdpr-info.eu/art-34-gdpr/) of GDPR). All the SQL queries to the database are logged in Acra by [AcraCensor]({{< ref "/acra/security-controls/sql-firewall/" >}}) (Acra&#39;s firewall) with configurable verbosity.
```

Comment:
- Split sentences into shorter chunks for easier reading. Adjusted links for easier clicking.
- Expanded a bit on the audit logging.
- Corrected wording about compliance. Acra by itself does not notify anyone about anything, its the administrator’s responsibility, which Acra merely assists with.
- Entities. Grammar.

```suggestion
All components of Acra continuously log requests, queries, and events that take place.
Acra also supports [secure logging]({{< ref "/acra/security-controls/security-logging-and-events/" >}})
which prevents tampering with, removing, adding, or changing the order of log entries.
This provides audit record of everything that happens in Acra, which comes particularly useful in case of a security incident.
Having a tamper-proof complete audit log makes it easier to notify the supervisory authority and users about the exact details of a data breach
(pursuant to [articles 33](http://gdpr-info.eu/art-33-gdpr/) [and 34](http://gdpr-info.eu/art-34-gdpr/) of GDPR).
All SQL queries to the database are logged by [AcraCensor]({{< ref "/acra/security-controls/sql-firewall/" >}}) (Acra’s firewall) with configurable verbosity.
```